docs/cep/query-guide/functions/geospatial/stationary.md

```diff
@@ -25,8 +25,21 @@ This function will return `true` when the object (defined in terms of longitude
 ## Example 1
 
 ```sql
-@info(name = 'query1')
-geo:stationary(km-4354,0,0, 110574.61087757687)
+geo:stationary('km-4354', 0, 0, 110574.61087757687)
 ```
 
-This example returns `true` because the given geofence with ID `km-4354` is considered stationary within the given radius of `110574.61087757687` from the reference point `(0, 0)`.
+In this example, the `geo:stationary()` function checks if the given geofence with ID `km-4354` is considered stationary within the given radius of `110574.61087757687` from the reference point `(0, 0)`. The function returns `true` if the geofence is stationary.
+
+## Example 2
+
+```sql
+CREATE STREAM InputGeoStream (geofenceId string, longitude double, latitude double, radius double);
+CREATE STREAM OutputGeoStream (stationaryStatus bool);
```

Comment:
@diana-macrometa - The second stream must be explicitly defined as `SINK` otherwise it's `SOURCE` by default.